detekt.yaml

```diff
@@ -1,5 +1,7 @@
 # See https://github.com/detekt/detekt/blob/master/detekt-core/src/main/resources/default-detekt-config.yml
 # for default settings.
+build:
+  excludeCorrectable: true
```

Comment:
Avoid failing on issues which can be auto corrected (only makes sense on local builds I guess)